docs/env.md

```
@@ -129,114 +129,134 @@ Environmental variables are also tracked in `ENVIRONMENT_VARIABLES` within `src/
-  - **minJobDuration**: Minimum duration in seconds for a free job.
-  - **maxJobs**: Maximum number of simultaneous free jobs.
-  - **allowImageBuild**: If building images is allowed on free envs. Default: false
-  - **access**: Access control configuration for free compute jobs. Works the same as the main `access` field.
-    - **addresses**: Array of Ethereum addresses allowed to run free compute jobs.
-    - **accessLists**: Array of AccessList contract addresses for free compute access control.
-  - **resources**: Array of resources available for free jobs.
-    - **id**: Resource type (e.g., `cpu`, `ram`, `disk`).
-    - **total**: Total number of the resource available.
-    - **min**: Minimum number of the resource needed for a job.
-    - **max**: Maximum number of the resource for a job.
+#### Connection-level fields
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)